CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added Guzzle and Symfony client factories.
```

Comment:
Added missing PR numbers in https://github.com/opensearch-project/opensearch-php/pull/290.